### PR TITLE
  fix nil value for giveitem

### PR DESCRIPTION
**Pull request by bamablood94**
**Describe Pull request**
Fixes Nil Value given in console after using /giveitem command with a missing 3rd arguement.

When you do /giveitem id item amount, if you dont specify the amount of items then you get a nil value error in console/chat. This fixes that

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? YES
- Does your code fit the style guidelines? YES
- Does your PR fit the contribution guidelines? YES

### CODE REVIEW

**Comment by bamablood94** [comment]
Alright, this sould be a fresh branch with just the one change

**Comment by TonybynMp4** [comment]
tbh could've just changed the 
`		if amount > 0 then
`
to
`		if amount and amount > 0 then
`

**Comment by jnccloud** [comment]
> **Describe Pull request** Fixes Nil Value given in console after using /giveitem command with a missing 3rd arguement.
> 
> When you do /giveitem id item amount, if you dont specify the amount of items then you get a nil value error in console/chat. This fixes that
> 
> **Questions (please complete the following information):**
> 
> * Have you personally loaded this code into an updated qbcore project and checked all it's functionality? YES
> * Does your code fit the style guidelines? YES
> * Does your PR fit the contribution guidelines? YES

This is a much more efficient and user friendly way to do this:

```lua
local amount = tonumber(args[3]) or 1
```

If its nil then just set it to 1 to make it more user friendly. Only 5 extra characters :)

File: server/main.lua [hunk 1]
```diff
@@ -1479,45 +1479,49 @@ QBCore.Commands.Add("giveitem", "Give An Item (Admin Only)", {{name="id", help="
 	local amount = tonumber(args[3])
```
Comment (jnccloud) [inline server/main.lua:1479]: You can undo all of your changes and change this line to do it in 5 characters:
```lua
local amount = tonumber(args[3]) or 1
```

File: server/main.lua [hunk 2]
```diff
@@ -1479,45 +1479,49 @@ QBCore.Commands.Add("giveitem", "Give An Item (Admin Only)", {{name="id", help="
...
+						info.lab = exports["qb-methlab"]:GenerateRandomLab()
+					elseif itemData["name"] == "printerdocument" then
+						info.url = "https://cdn.discordapp.com/attachments/870094209783308299/870104331142189126/Logo_-_Display_Picture_-_Stylized_-_Red.png"
+					end
+	
+					if Player.Functions.AddItem(itemData["name"], amount, false, info) then
+						TriggerClientEvent('QBCore:Notify', source, "You Have Given " ..GetPlayerName(tonumber(args[1])).." "..amount.." "..itemData["name"].. "", "success")
+					else
+						TriggerClientEvent('QBCore:Notify', source,  "Can't give item!", "error")
+					end
```
Comment (jnccloud) [inline server/main.lua:1516]: Revert _all_ of this

File: server/main.lua [hunk 3]
```diff
@@ -1479,45 +1479,49 @@ QBCore.Commands.Add("giveitem", "Give An Item (Admin Only)", {{name="id", help="
...
-					TriggerClientEvent('QBCore:Notify', source,  "Can't give item!", "error")
+					TriggerClientEvent('QBCore:Notify', source,  "Item Does Not Exist", "error")
 				end
 			else
-				TriggerClientEvent('QBCore:Notify', source,  "Item Does Not Exist", "error")
+				TriggerClientEvent('QBCore:Notify', source,  "Invalid Amount", "error")
 			end
 		else
-			TriggerClientEvent('QBCore:Notify', source,  "Invalid Amount", "error")
+			TriggerEvent('QBCore:Notify', source, 'Missing Arguement', 'error')
```
Comment (jnccloud) [inline server/main.lua:1524]: All of these notifications can be reverted too if you use my method

**Review by jnccloud — CHANGES_REQUESTED** [review]
Can be refactored to be more user friendly with less code.

**Comment by jnccloud** [comment]
This was actually already fixed in: 

https://github.com/qbcore-framework/qb-inventory/commit/2e042e69e67f5fe91fc8bf094adcdf8a05900e26

This should be closed..

**Pull request closed without merging**